### PR TITLE
some optimization

### PR DESCRIPTION
**Pull request by Ryan-Git**

### CODE REVIEW

**Comment by ming535** [comment]
we can delete this file now `pkg/debug.go`

File: pkg/schema_store/simple_schema_store.go [hunk 1]
```diff
@@ -23,21 +23,35 @@ func (store *SimpleSchemaStore) IsInCache(dbName string) bool {
 		return false
 	}
 }
+func (store *SimpleSchemaStore) getFromCache(dbName string, lock bool) (Schema, bool) {
+	if lock {
+		store.RLock()
```
Comment (ming535) [inline pkg/schema_store/simple_schema_store.go:28]: how much improvement can we get from usign `RLock`?

File: pkg/sliding_window/static_sliding_window.go [hunk 2]
```diff
@@ -115,7 +115,10 @@ func (w *staticSlidingWindow) removeItemFromSequence() (WindowItem, bool) {
 			return nextItem, ok
 
 		case <-ticker.C:
-			w.reportWatermarkDelay()
+			delay := w.reportWatermarkDelay()
+			if delay > 30 {
```
Comment (ming535) [inline pkg/sliding_window/static_sliding_window.go]: define `30` as a const

**Review by ming535 — CHANGES_REQUESTED** [review]
we can delete this file now `pkg/debug.go`

File: pkg/schema_store/simple_schema_store.go (hunk 1 above)
Comment (Ryan-Git) [reply pkg/schema_store/simple_schema_store.go:28]: From the mutex profiler, the delay dropped from 63.12% to 16.87%. And it's the only delay under our control. The other one is from sql.DB.conn.

File: pkg/sliding_window/static_sliding_window.go (hunk 2 above)
Comment (Ryan-Git) [reply pkg/sliding_window/static_sliding_window.go]: done

**Review by ming535 — APPROVED** [review]

**Pull request merged**